### DISCUSSION TITLE
Laravel Sail - Option to change the ubuntu source / China

### DISCUSSION THREAD

**Discussion by cstriuli** [laravel/framework#36400] [Q&A]
Is it possible to add an option to `laravel/sail` to download the Ubuntu source from the anothers servers? Like aliyun for China. 
I'm trying to customize the `Dockerfile` that sails creates but in order to publish the vendor files I've to wait for `sails up` to finish but it takes hours and fails in the end. 

Here you can find a PR that merged in laradock (laravel docker image):
https://github.com/laradock/laradock/pull/2388/files

**Comment by carveybunt** [comment]
我也是想从 laradock 转 sail 貌似条件不允许哦。看到您再 sail 哪边也发表了。
